CHANGELOG.md

```diff
@@ -53,6 +53,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Improved handling for receiving SIGTERM during training. By default, SIGTERM triggers 'save (now) and exit'. Prior to this fix, batch pre-fetching did not check for this sigal, potentially delaying exit considerably. It now pays attention to that. Also, the default behaviour of save-and-exit can now be disabled on the command line with --sigterm exit-immediately.
 
 ### Changed
+- Updated SentencePiece repository to version 8336bbd0c1cfba02a879afe625bf1ddaf7cd93c5 from https://github.com/google/sentencepiece. Enabled compilation by default since no dependency on protobuf anymore. Changed default value of --sentencepiece-max-lines to from 10000000 to 2000000 since apparently the new version doesn't sample automatically anymore. Not quite clear how that affects quality of the vocabulary.
```

Comment:
"to from" -> "to" + I would suggest making a separate change including the second and third sentences.

Reply:
Not quite sure I understand. Do you mean to have a separate bullet point for the 2. and 3. sentence? They do come from the same PR though?

Reply:
I mean that there are two changes: 1) updating SentencePiece, 2) changing the default value of an option; so maybe they deserve separate bullets. That's a minor comment.

Reply:
Understood and changed accordingly.